Changelog.md

```diff
@@ -20,6 +20,8 @@
 - Adds config option `profile.*.fonts.TYPE.weight: WEIGHT` and `profile.*.fonts.TYPE.slant: SLANT` options (optional) along with `profile.*.fonts.TYPE.family: STRING`.
 - Adds terminal identification environment variables `TERMINAL_NAME`, `TERMINAL_VERSION_TRIPLE` and `TERMINAL_VERSION_STRING`.
 - Adds config option `profile.*.fonts.TYPE.weight: WEIGHT` and `profile.*.fonts.TYPE.slant: SLANT` options (optional) along with `profile.*.fonts.TYPE.family: STRING`.
+- Adds VT sequence `CSI 18 t` and `CSI 19 t` for getting screen character size. Responds with `CSI 8 ; <columns> ; <rows> t` and  `CSI 9 ; <columns> ; <rows> t` respectively.
+- Adds VT sequence to capture the current screen buffer `OSC 314 ; Mode ; StartLine ; LineCount ST` giving the respone back on stdin via `OSC 314 ; <screen buffer> ST`, and feature detection via `DA1` number `314`.
```

Comment:
Response on STDIN?

Reply:
yes, so in the shell you can use the `read` command and in C/C++ the `int read(int fd, char* targetBuf, size_t bufSize)` syscall with `fd` being `STDIN_FILENO` (= `1`).